### ISSUE TITLE
Bug with test_valid_entry in integrity tests.

### ISSUE THREAD

**Issue by souravtecken** [varunvora/alcoding#22]
The test will always pass, unless it's an invalid username.
An invalid USN will pass as a valid username.

**Comment by varunvora** [comment]
Yes, I agree. I don't think it is a bug, is there any other way of testing it?

**Comment by souravtecken** [comment]
I don't think we can do anything unless the databse is final, because there still are unmapped handles in the codechef rank files. After we're sure we have everybody, then I think we can work on it?
